src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -409,15 +409,23 @@ ValidatorList::for_each_listed (
 }
 
 std::size_t
-ValidatorList::calculateQuorum (std::size_t nTrustedKeys)
+ValidatorList::calculateMinimumQuorum (
+    std::size_t nListedKeys, bool unlistedLocal)
 {
-    // Use 80% for large values of n, but have special cases for small numbers.
-    constexpr std::array<std::size_t, 10> quorum{{ 0, 1, 2, 2, 3, 3, 4, 5, 6, 7 }};
+    // Use 2f + 1 for large values of n, but have special cases for small numbers.
+    constexpr std::array<std::size_t, 6> quorum{{ 0, 1, 2, 2, 3, 3 }};
```

Comment:
That ideone calculation is doing `i/2 + 1` as opposed to `2/3 * i + 1`.
Are you suggesting that we calculate a 51% quorum for smaller values as opposed to using the `std::array`? I see how that would work.